index.html

```diff
@@ -0,0 +1,533 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <script src="https://kit.fontawesome.com/5b1eeceadd.js" crossorigin="anonymous"></script>
+  <title>TNW-Website-Clone</title>
+  <link rel="stylesheet" href="./css/style.css">
+</head>
+
+<body>
+  <header>
+    <nav class="nav">
+      <div class="nav-logo">
+        <a href="index.html"><img src="./images/tnw-red.svg" alt=""></a>
+        <a href=""><i class="fas fa-bars bar fa-2x"></i></a>
+      </div>
+      <div class="lists">
+        <div class="list-container">
+          <ul>
+            <li><a href="">News</a></li>
+            <li><a href="">Events</a></li>
+            <li><a href="">Business</a></li>
+            <li><a href="">AMAs</a></li>
+            <li><a href="">Spaces</a></li>
+            <li><a href="">Terms&Conditions</a></li>
+            <li><a href="">About</a></li>
+            <li><a href="">Advertise</a></li>
+            <li><a href="">Jobs</a></li>
+            <li><a href="">Contact</a></li>
+          </ul>
+        </div>
+        <div class="nav-list3">
+          <ul class="nav-section">
+            <li><a href="">LATEST</a></li>
+            <li><a href="">HARD FORK</a></li>
+            <li><a href="">PLUGGED</a></li>
+            <li><a href="">README</a></li>
+            <li><a href="">GROWTH QUARTERS</a></li>
+            <li><a href="">SHIFT</a></li>
+            <li><input type="text" name="search" id="search"></li>
+          </ul>
```

Comment:
- [ ] The page contains a search icon at the rightmost side of the sub-navigation row in the original large screen but the search icon is missing in your version. Please add it.

- [ ] Also the navigation and the sub-navigation items should be aligned left but in your version, there is a redundant empty space, please fix your CSS rules to create the same UI with the original page.

![image](https://user-images.githubusercontent.com/60184961/93028723-5690bf80-f61e-11ea-83c4-e8a41fc15cf1.png)